Source/BCfill.cpp

```diff
@@ -145,10 +190,86 @@ pc_bcfill_hyp(
   const int bcomp,
   const int scomp)
 {
-  const ProbParmDevice* lprobparm = PeleC::d_prob_parm_device;
-  amrex::GpuBndryFuncFab<PCHypFillExtDir> hyp_bndry_func(
-    PCHypFillExtDir{lprobparm});
+  ProbParmDevice* probparmDD = PeleC::d_prob_parm_device; // probparm data for device
+  ProbParmDevice* probparmDH = PeleC::h_prob_parm_device; // host copy of probparm data for device
+  ProbParmHost* probparmH = PeleC::prob_parm_host;        // probparm data for host
+  constexpr int dim = AMREX_SPACEDIM;
+
+#ifdef PELEC_USE_TURBINFLOW
+  if (probparmH->do_turb) {
```

Comment:
I am thinking maybe we move this block and the one after this one to their own functions. Just to keep things more readable?